HouseRules_Core/README.md

```diff
@@ -29,30 +37,61 @@ If installation instructions were followed (i.e., MelonLoader was installed),
 the following file will appear in the Demeo game directory:
 `UserData/MelonPreferences.cfg`.
 
+Demeo should not be running when you are editing this file or your changes may not be saved.
+
 After RulesAPI is installed and Demeo is run at least once, the following will
 appear somewhere in that configuration file:
 
 ```toml
-[RulesAPI]
+[HouseRules]
 ruleset = ""
+loadFromConfig = false
 ```
 
-Select the ruleset to use by typing its name within the quotes.  Alternatively,
-use empty quotes `""` to specify no ruleset should be used.
+A list of out-of-the-box Ruleset names can be found in the
+[Rules readme](../HouseRules_Essentials/README.md).
 
-For now, Demeo should not be running when the configuration file is modified.
-Else your changes may not be saved.
+To use a built-in ruleset, ensure `loadFromConfig` is set to `false` and select the built-in ruleset to use by typing its name within the quotes.
+Alternatively, use empty quotes `""` to specify no ruleset should be used.
+
+Rulesets may also be defined in JSON within `UserData/MelonPreferences.cfg`. Here's a ruleset for beefing up the Sorcerer's Zap ability.
+
+```toml
+[HouseRules]
+ruleset = "SuperZapRuleset"
+loadFromConfig = true
+
+[SuperZapRuleset]
+name = "SuperZapRuleset"
```

Comment:
No issue here, just a point for discussion:

It makes sense to have a suffix for `Rule` classes (e.g., `SampleRule.cs`), because of class naming practices.

For rulesets, however, as the names exist really just as a string, I don't think we ought to feel obligated to have the `Ruleset` suffix (outside of class naming).

100% just a side note. 👍🏾

Reply:
I just called the ruleset 'SuperZap' in the latest commit.